### PR TITLE
fix(server): message broker deletion referenced data  

### PR DESCRIPTION
**Pull request by overbit**
Close: #5303
## PR Details

Update message broker deletion process to delete all relative reference blocks

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-server/project.json [hunk 1]
```diff
@@ -72,6 +72,7 @@
     "serve": {
       "executor": "@nrwl/node:node",
       "options": {
+        "port": 7000,
```
Comment (abrl91) [inline packages/amplication-server/project.json:75]: what is that for?

File: packages/amplication-server/src/core/resource/resource.service.ts [hunk 2]
```diff
@@ -386,8 +424,12 @@ export class ResourceService {
       throw new Error(INVALID_RESOURCE_ID);
     }
 
-    if (resource.resourceType === EnumResourceType.ProjectConfiguration) {
-      throw new Error(INVALID_DELETE_PROJECT_CONFIGURATION);
+    switch (resource.resourceType) {
+      case EnumResourceType.ProjectConfiguration:
+        throw new Error(INVALID_DELETE_PROJECT_CONFIGURATION);
```
Comment (abrl91) [inline packages/amplication-server/src/core/resource/resource.service.ts:429]: I think @MichaelSolati worked on project deletion, which, if I'm not mistaken, is basically deleted to project configuration.

Comment (overbit) [reply packages/amplication-server/src/core/resource/resource.service.ts:429]: Not changes have been made here apart from a minor refactoring.

File: packages/amplication-server/project.json (hunk 1 above)
Comment (overbit) [reply packages/amplication-server/project.json:75]: debugging port. This in conjunction with the attach debugging server launch config, will let us debug the serve app,

**Review by abrl91 — APPROVED** [review]

**Review by morhag90 — APPROVED** [review]

**Pull request merged**